packages/@apphosting/adapter-nextjs/src/interfaces.ts

```diff
@@ -101,3 +101,11 @@ export interface OutputBundleOptions {
    */
   outputStaticDirectoryPath: string;
 }
+
+// Metadata schema for bundle.yaml outputted by next.js adapter
+export interface Metadata {
+  adapterNpmPackageName: string;
```

Comment:
ooc how does this change when we support non node dependencies, eg for python apps?

Reply:
We can modify the output bundle spec naming when needed? Revisiting in discussion meeting.

Reply:
Discussed and agreed to make this field more generic, changing to`adapterPackageName`